### PR TITLE
Add warning about scoped Slot implementation

### PR DESCRIPTION
**Pull request by LinusBorg**

### CODE REVIEW

File: docs/api/options.md [hunk 1]
```diff
@@ -124,6 +124,28 @@ shallowMount(Component, {
   }
 })
 ```
+::: warning Root Element required
+Due to the internal implementatioj of this feature, the slot content has to return a root element, even though a scoped slot is allowed to return an array of elements.
```
Comment (wiese) [inline docs/api/options.md]:
```suggestion
Due to the internal implementation of this feature, the slot content has to return a root element, even though a scoped slot is allowed to return an array of elements.
```

**Comment by eddyerburgh** [comment]
Thanks @LinusBorg . Can you run prettier to fix the formatting issue?

**Comment by LinusBorg** [comment]
@eddyerburgh 

* when i run prettier on the whole repo (`yarn format`) it changes >200 files.
* When I run it on just the docs file (yarn prettier --no-semi --single-quote --write "./docs/api/options"`) it also changes around 20 other things, mostly turning `- ` into `* ` for lists.

No idea why?

**Comment by winniehell** [comment]
@LinusBorg Just a wild guess but maybe https://github.com/vuejs/vue-test-utils/pull/1247 helps here. :thinking:

**Comment by lmiller1990** [comment]
I tried to rebase and encountered the same problem described above, but did not notice until pushing the commits. So I just made a fresh fork of dev and copied the content over. It's here: #1389.

Going to close this one and get the other one merged.

**Pull request closed without merging**